src/main/java/com/reimbursements/reporitories/TypeRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.reimbursements.reporitories;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import com.reimbursements.models.Type;
+
+public interface TypeRepository extends JpaRepository<Type, Integer> {
+	public Type getByTypeID(int typeID);

```

Comment:
This method is not required. By default, `JpaRepository` implements some basic methods: get by ID is one of them.